src/main/java/nextstep/subway/domain/StationShortestPath.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.subway.domain;
+
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+import java.util.List;
+
+public class StationShortestPath {
+
+    private DijkstraShortestPath dijkstraShortestPath;
```

Comment:
재할당이 필요없다면 final 을 사용해보면 어떨까요? 😄 
```suggestion
    private final DijkstraShortestPath dijkstraShortestPath;
```

Reply:
아 자꾸 이런 실수를 하네요;;
바로 반영하겠습니다